package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/hashgraph/hedera-smart-contracts#readme",
   "devDependencies": {
-    "@hashgraph/hedera-local": "^2.8.0",
+    "@hashgraph/hedera-local": "2.11.3",
```

Comment:
```suggestion
    "@hashgraph/hedera-local": "2.13.0",
```

Reply:
Done.

Reply:
Doesn't look like the change was pushed.

Reply:
Yes, the package.json file was not staged. It's been committed and pushed. Apologies.